content/tutorials/earthworks/levees.qmd

```diff
@@ -0,0 +1,616 @@
+description: Model levees and alluvial ridges with r.earthworks.
+image: images/levees_08.webp
+links:
+  r_earthworks: "[r.earthworks](https://grass.osgeo.org/grass-stable/manuals/addons/r.earthworks.html)"
+  g_extension: "[g.extension](https://grass.osgeo.org/grass-stable/manuals/g.extension.html)"
+  g_region: "[g.region](https://grass.osgeo.org/grass-stable/manuals/g.region.html)"
+  r_relief: "[r.relief](https://grass.osgeo.org/grass-stable/manuals/r.relief.html)"
+  d_shade: "[d.shade](https://grass.osgeo.org/grass-stable/manuals/d.shade.html)"
+  r_lake: "[r.lake](https://grass.osgeo.org/grass-stable/manuals/r.lake.html)"
+  r_mapcalc: "[r.mapcalc](https://grass.osgeo.org/grass-stable/manuals/r.mapcalc.html)"
+  v_transform: "[v.transform](https://grass.osgeo.org/grass-stable/manuals/v.transform.html)"
+code-links:
+  - text: Jupyter notebook
+    href: levees.ipynb
+    icon: file-code
+    target: _blank
+resources: levees.ipynb
+format:
+  html:
+    toc: true
+    code-tools: true
+    code-copy: true
+    code-fold: false
+engine: jupyter
+execute:
+  eval: false
+jupyter: python3
+---
+
+![Levee improvement & ridge restoration](images/levees_08.webp)
+
+Learn the how to model levees and ridges with 
+{{< meta links.r_earthworks >}}.
+Download the [Bayou L'Ours Dataset](https://doi.org/10.5281/zenodo.15870441).
+The coordinate reference system for this project is
+Louisiana State Plane South in meters.
+We will model two of the coastal infrastructure projects 
+in the Louisiana Coastal Protection and Restoration Authority's 
+[2023 Coastal Masterplan](https://coastal.la.gov/wp-content/uploads/2023/06/230531_CPRA_MP_Final-for-web_spreads.pdf). 
+Improvements are planned to the 
+[Larose to Golden Meadow Levee System](https://coastal.la.gov/wp-content/uploads/masterplan/111_Larose-to-Golden-Meadow.pdf)
+which will raise existing earthen levees to an elevation between 3.6-6.4 meters NAVD88.
+The [Bayou L'Ours Ridge Restoration](https://coastal.la.gov/wp-content/uploads/masterplan/334_Bayou-LOurs-Ridge-Restoration.pdf) 
+project will restore the natural levee by
+closing gaps, rebuilding landforms, and reforesting uplands. 
+While the improved levee system will better protect the towns,
+the restored ridge will attenuate storm surge.
+This tutorial covers:
+
+* Visualizing existing conditions
+* Modeling ridge restoration
+* Modeling levee improvement
+* Modeling gap closure
+* Simulating flooding
+
+::: {.callout-note title="Computational notebook"}
+This tutorial can be run as a 
+[computational notebook](https://grass-tutorials.osgeo.org/content/tutorials/earthworks/levees.ipynb).
+Learn how to work with notebooks in the tutorial
+[Get started with GRASS & Python in Jupyter Notebooks](./get_started/fast_track_grass_and_python.qmd).
+:::
+
+# Setup
+
+## Project
+
+Start a GRASS session in the Bayou L'Ours project. 
+
+::: {.panel-tabset group="language"}
+
+## Command line
+
+```{bash}
```

Comment:
Note that the CLI still needs the PERMANENT mapset there.
```suggestion
```{bash}
grass ~/grassdata/bayou_lours/PERMANENT
```